### DISCUSSION TITLE
ohmyzsh changes pid 1 from systemd to init

### DISCUSSION THREAD

**Discussion by wangjian1106** [ohmyzsh/ohmyzsh#11173] [Help] [answered]
I've install [wsl-distrod](https://github.com/nullpo-head/wsl-distrod) into wsl2 on my windows 10.  
The pid 1 is `systemd`. 
```bash
ps -p 1 -o comm=
```
After zsh and ohmyzsh installed, open a new terminal, run `ps -p 1 -o comm=`, i got `init`.
Is that possible to keep pid 1 `systemd` ?

**Comment by carlosala** [comment, marked as answer]
This doesn't depend at all of zsh!

**Reply by wangjian1106** [reply]
Hello @carlosala , Thanks for your replying.  
Could you please tell me what does it depend at ? i've googled and did not get any answers.

**Reply by carlosala** [reply]
I'm not an expert about that, I can't provide you support. Sorry!

**Reply by wangjian1106** [reply]
It's ok. thanks.
